scripts/build/git-id

```diff
@@ -32,9 +32,17 @@ get_gitids () {
             fi
         fi
     else
-        DIR=${DIR:-.}
-        version=$(cat $DIR/git-version || unknown_version)
-        buildid=$(cat $DIR/git-buildid || echo unknown)
+        DIR=${DIR:-$(pwd)}
```

Comment:
There's a bunch of shellcheck warnings for this... `$DIR` should be in double quotes here and elsewhere to handle potential spaces etc. in directory names.